standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStore.java

```diff
@@ -1856,6 +1859,49 @@ public void testColumnStatistics() throws Throwable {
       colStats2 = client.getTableColumnStatistics(
           dbName, tblName, Lists.newArrayList(colName[0]), ENGINE).get(0);
 
+      // test delete column stats; if no col name is passed all column stats associated with the
+      // table is deleted
+      status = client.deleteTableColumnStatistics(dbName, tblName, colName[0], ENGINE);
+      assertTrue(status);
+      // try to query stats for a column for which stats doesn't exist
+      stats = client.getTableColumnStatistics(
+              dbName, tblName, Lists.newArrayList(colName[0]), ENGINE);
+      assertTrue("stats are not empty: " + stats, stats.isEmpty());
+
+      // reset the column stats
+      client.updateTableColumnStatistics(colStats);
+      // test delete multiple column stats(column names can be null, or one, or multiple)
+      // case 1: column names are null, all column stats should be deleted
+      status = client.deleteTableMultiColumnStatistics(dbName, tblName, null, ENGINE);
+      assertTrue(status);
+      stats = client.getTableColumnStatistics(
+              dbName, tblName, Lists.newArrayList(colName[0]), ENGINE);
+      assertTrue("stats are not empty: " + stats, stats.isEmpty());
+      stats = client.getTableColumnStatistics(
+              dbName, tblName, Lists.newArrayList(colName[1]), ENGINE);
+      assertTrue("stats are not empty: " + stats, stats.isEmpty());
+
+      client.updateTableColumnStatistics(colStats);
```

Comment:
we have tested the single column case at https://github.com/apache/hive/pull/5578/files#diff-120e8efba348e9a8c233de4135cdcc4aaeb6bc0fe2540d9d952554af2c05fdecR1867

Reply:
I am not sure if I understand this.
In this test file, we are testing 3 cases. (Columns names amount are null, 1, or multiple)